examples/advanced_operations/add_sitelink.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Demonstrates how to create a sitelink extension."""
+
+from __future__ import absolute_import
+
+import argparse
+import six
+import sys
+import uuid
+
+import google.ads.google_ads.client
```

Comment:
Can you update this to:

```
from google.ads.google_ads.client import GoogleAdsClient
```

And on line 59 you can just use:

```
google_ads_client = GoogleAdsClient.load_from_storage()
```

Reply:
much better, we should probably change that in the other examples as well.